### ISSUE TITLE
train a topic classifier on English MultiEURLEX

### ISSUE THREAD

**Issue by lannelin** [alan-turing-institute/ARC-SPICE#23]
- must be able to load from huggingface for compatibility with pipeline
- must use dropout at training

**Comment by lannelin** [comment]
model on baskerville: `[/path/to/project]/distilbert-topic-classifier`

training config can be found in the above folder but highlights:
epochs: 5
learning rate: 5e-5
batch size: 128
